Modules/posixmodule.c

```diff
@@ -1261,7 +1261,7 @@ _Py_Sigset_Converter(PyObject *obj, void *addr)
     long signum;
     int overflow;
 
-    if (sigemptyset(mask)) {
+    if (sigemptyset(mask) < (0)) {
```

Comment:
why the parentheses?

Reply:
It's the way to disable it with clang:
```
./Modules/posixmodule.c:1266:9: warning: code will never be executed [-Wunreachable-code]
        PyErr_SetFromErrno(PyExc_OSError);
        ^~~~~~~~~~~~~~~~~~
./Modules/posixmodule.c:1264:29: note: silence by adding parentheses to mark code as explicitly dead
    if (sigemptyset(mask) < 0) {
                            ^
                            /* DISABLES CODE */ ( )
```

Reply:
Maybe I should put a comment about it?

Reply:
Yes, please.

Reply:
Thanks @methane, I added a comment about it